ui/components/ui/box/box.js

```diff
@@ -72,10 +72,29 @@ export const BorderColors = [
   COLORS.LOCALHOST,
 ];
 
+export const Colors = [
+  COLORS.TEXT_DEFAULT,
+  COLORS.TEXT_ALTERNATIVE,
+  COLORS.TEXT_MUTED,
+  COLORS.OVERLAY_INVERSE,
+  COLORS.PRIMARY_DEFAULT,
+  COLORS.PRIMARY_INVERSE,
+  COLORS.SECONDARY_DEFAULT,
+  COLORS.SECONDARY_INVERSE,
+  COLORS.ERROR_DEFAULT,
+  COLORS.ERROR_INVERSE,
+  COLORS.SUCCESS_DEFAULT,
+  COLORS.SUCCESS_INVERSE,
+  COLORS.WARNING_INVERSE,
+  COLORS.INFO_DEFAULT,
+  COLORS.INFO_INVERSE,
+];
+
```

Comment:
I dont' think we should be declaring another COLORS constant here... If the main colors constant contains colors that are invalid for the Box component, use lodash's pick, or omit method. Duplicating that effort in the stories is worthwhile to avoid the confusion of multiple Colors constants in the code base. Also hopefully we work towards removing any unsupported colors from the Colors constant in design-system so that we no longer have to specify.

Reply:
@brad-decker I think we should replace `colors` with `TextColors` to avoid confusion. Another thing is I have used the colors in this object which are the same as what we have in Typography. So, I am not sure if we should use the pick or omit method since it will also create a similar object for us.

Reply:
@NidhiKJha I like the idea of consolidating these to  an *object* called TEXT_COLORS that we can use in the same way as COLORS in the design-system.js file. This way we can avoid declaring and exporting these in the component files themselves.

Reply:
Ooo yes please! Can we do this for the BackgroundColors => `BACKGROUND_COLORS` and BorderColors => `BORDER_COLORS` too? We can also update the `Typography` component to use them as well

Reply:
Updated!